src/Symfony/Component/Console/Style/SymfonyStyle.php

```diff
@@ -159,6 +167,14 @@ public function error($message)
     /**
      * {@inheritdoc}
      */
+    public function errorLine($message)
```

Comment:
I'll change their name...

Maybe to get what I'm trying to accomplish I should edit `OutputFormatter`...